### PR TITLE
syscall: remove obsolete syscalls

### PR DESCRIPTION
**Pull request by davecheney**
Update #91
- Update mksysnum and zsysnum, remove obsolete syscall numbers
- Replace obsolete syscalls with forwarding methods
- For the remaining calls, temporarily move their definition to xsyscall_linux_arm64.go

### CODE REVIEW

File: src/syscall/mksysnum_linux.pl [hunk 1]
```diff
@@ -18,6 +18,10 @@ package syscall
 
 sub fmt {
 	my ($name, $num) = @_;
+	if($num > 999){
```
Comment (minux) [inline src/syscall/mksysnum_linux.pl:21]: add a reference to https://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/tree/include/uapi/asm-generic/unistd.h?id=refs/heads/master#n716
to explain the meaning of 999.

also I think we should limit this only to arm64.

Comment (davecheney) [inline src/syscall/mksysnum_linux.pl:21]: I'll do that in my next patch set on this PR

**Comment by davecheney** [comment]
PTAL. This should fix the problems you were having with open(2) and stat(2). 

You can merge this and I'll do the remaining syscalls in a followup.

**Comment by minux** [comment]
LGTM. Thanks!

**Comment by davecheney** [comment]
@4ad do you want to merge what is here and i'll pick up the rest tomorrow on a new PR ?

**Pull request merged**